codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/ConstraintViolationSymbolProvider.kt

```diff
@@ -117,6 +118,20 @@ class ConstraintViolationSymbolProvider(
                     .definitionFile(Models.filename)
                     .build()
             }
+            is IntegerShape -> {
```

Comment:
We should make this DRYer given that the arm is identical to `StringShape`. See how `constraintViolationSymbolForCollectionOrMapOrUnionShape` is used.

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2005/commits/72cda3a82631a88dfba9caefea450547f3d91fa3